src/lib/dongle/mod.rs

```diff
@@ -194,9 +194,11 @@ pub trait Dongle {
             });
         }
 
-        if rev != [0, 0] {
-            panic!("Ledger requested user authentication but we don't know how to handle that");
-        }
+        assert_ne!(
```

Comment:
yeah, I think the logic ended up reversed.  Before we failed if `rev != [0, 0]` but now we are asserting that `rev != [0, 0]` is required.

Reply:
Lol, yeah. Thanks for testing! Sorry for pushing this up without trying it.